README.md

```diff
@@ -7,6 +7,9 @@ and the [hyper HTTP library](https://github.com/hyperium/hyper).
 [![Documentation](https://docs.rs/hyper-rustls/badge.svg)](https://docs.rs/hyper-rustls/)
 
 # Release history
+
+Newer releases can be found [on GitHub](https://github.com/rustls/hyper-rustls/releases).
```

Comment:
```suggestion
Release history can be found [on GitHub](https://github.com/rustls/hyper-rustls/releases).
```

Let's delete the following older release history -- I'll move that text into github releases so we have this in one place.

Reply:
Fixed.